src/50_Sample_AMPs/Sports_Article.html

```diff
@@ -0,0 +1,115 @@
+<!--
+  #### Introduction
+
+This is a sample article demonstrating how to format a sports article relating to a particular sporting event with AMP. It demonstrates how to use schema.org structured markup to apply the correct meta-data to describe the sporting event the article is about.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+  <head>
+    <meta charset="utf-8">
+    <link rel="canonical" href="https://www.ampbyexample.com/sample_amps/sports_article" />
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <!--
+    #### Metadata
+    A sports article should use the schema.org markup for the NewsArticle. When reporting on a specific sporting event, the NewsArticle should be extended to include the specific sporting event the article is about (see the "about" field)
+    -->
+    <script type="application/ld+json">
+    {
+      "@context": "http://schema.org",
+      "@type": "NewsArticle",
+      "mainEntityOfPage": "https://www.ampbyexample.com/sample_amps/sports_article",
+      "headline": "The Toronto Wildcats defeat the San Jose Stars 4-2",
+      "url": "https://www.ampbyexample.com/sample_amps/sports_article",
+      "datePublished": "2016-04-01T12:00:00Z",
+      "dateModified": "2016-04-01T12:00:00Z",
+      "description": "The Toronto Wildcats defeat the San Jose Stars 4-2",
```

Comment:
What do you think about making headline different than the description? "Rivals face off during biggest game of the season."